src/include/ipc4/module.h

```diff
@@ -26,6 +26,8 @@
 
 #include <stdint.h>
 
+#define IPC4_MAX_SUPPORTED_LIBRARIES 16
```

Comment:
why 16?

Reply:
this copied from user mode audio sdk for customers

Reply:
Can this also have a TODO revisit comment as we will want to remove any restriction.